CHANGELOG.md

```diff
@@ -11,10 +11,20 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
+- Add gRPC trace `Filter` for stats handler to `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc`. (#5196)
+
+### Changed
+
+- The gRPC trace `Filter` for interceptor is renamed to `InterceptorFilter`. (#5196)
+- The gRPC trace filter functions `Any`, `All`, `None`, `Not`, `MethodName`, `MethodPrefix`, `FullMethodName`, `ServiceName`, `ServicePrefix` and `HealthCheck` for interceptor are moved to `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc/filters/interceptor`. (#5196)
+
+### Deprecated
```

Comment:
The OTEL_METRICS_PRODUCERS isn't Deprecated.  Can you keep existing CHANGELOG entries in their original category?

Reply:
Thanks for the catch, fixed with https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5196/commits/733f3be58486734542d38bc5612004c598dd1764